netavark.spec.rpkg

```diff
@@ -58,12 +60,25 @@ popd
 %install
 %{__make} DESTDIR=%{buildroot} PREFIX=%{_prefix} install
 
+%post
+if [ $1 -eq 1 ]; then
+    %{_bindir}/systemctl enable --now %{name}-dhcp-proxy.service
+fi
+
+%preun
+%systemd_preun %{name}-dhcp-proxy.service
+
+%postun
+%systemd_postun %{name}-dhcp-proxy.service
+
 %files
 %license LICENSE
 %dir %{_libexecdir}/podman
 %{_libexecdir}/podman/%{name}
 %{_libexecdir}/podman/%{name}-dhcp-proxy
 %{_mandir}/man1/%{name}.1*
+%{_unitdir}/%{name}.service
+%{_unitdir}/%{name}.socket
```

Comment:
these need to be `%{name}-dhcp-proxy` instead of just `%{name}`. 

Right now the packit rpm builds are failing with:
```
RPM build errors:
    source_date_epoch_from_changelog set but %changelog is missing
    File not found: /builddir/build/BUILDROOT/netavark-1.6.0~dev-1.20230314174247870645.pr620.58.g722fa8a.fc39.x86_64/usr/lib/systemd/system/netavark.service
    File not found: /builddir/build/BUILDROOT/netavark-1.6.0~dev-1.20230314174247870645.pr620.58.g722fa8a.fc39.x86_64/usr/lib/systemd/system/netavark.socket
```

Reply:
Thank you for catching this 🙂